test/logging_test.go

```diff
@@ -0,0 +1,131 @@
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/ministryofjustice/cloud-platform-infrastructure/test/helpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Logging tests define the ability for Cloud Platform to perform aggregated logging
+// on the platform. The tests are designed to be run in a Kubernetes cluster, with a logging agent installed.
+var _ = Describe("logging", func() {
+	Context("when an app generates a log message", func() {
+		var (
+			namespace string
+			options   *k8s.KubectlOptions
+		)
+
+		var (
+			elasticSearch = "https://search-cloud-platform-live-dibidbfud3uww3lpxnhj2jdws4.eu-west-2.es.amazonaws.com"
+			date          = time.Now().Format("2006.01.02")
+			search        = elasticSearch + "/live_kubernetes_cluster" + "-" + date + "/_search"
+		)
+
+		BeforeEach(func() {
+			if !strings.Contains(c.ClusterName, "live") {
```

Comment:
This is opinionated, but is it worthwhile using `strings.HasPrefix(c.ClusterName, "live")` here, instead of `strings.Contains`?

I'm thinking about cases where a cluster name may be `non-live`, which would run this test.